docs/components/headings/README.md

```diff
@@ -75,151 +82,328 @@
 <template slot="Overview">
 <cdr-doc-table-of-contents-shell>
 
-## Display Responsive
+Stuart is our first choice for larger, more expressive moments. Graphik rounds out the heading schema with smaller sizes and its simplicity allows it to play a supporting role across a wide range of projects.
+
+**Usage Considerations**
+-  Heading styles should never end with a period
+-  Sentence case should be used whenever possible
+-  Consider the white space around headings
+
+## Contextual Headings
+
+The following are recommended type styles for use at varying breakpoint sizes.
 
-Use for responsive display heading.
+### Headline 
 
+Use for the most prominent type style on the page in place of titles. Also, use headings rarely to emphasize important information.
 
 <cdr-doc-example-code-pair repository-href="/src/components/text" :sandbox-data="$page.frontmatter.sandboxData" >
 
 ```html
   <cdr-text
-    tag="h4"
-    modifier="display">
+    tag="p"
```

Comment:
`p` is the default isn't it? We probably don't need to keep this as it will probably confuse people